protocol-designer/src/components/modals/FilePipettesModal/index.tsx

```diff
@@ -361,7 +367,7 @@ export const FilePipettesModal = (props: Props): JSX.Element => {
         // @ts-expect-error(sa, 2021-6-21): TODO validate that pipette names coming from the modal are actually valid pipette names on PipetteName type
         return formPipette &&
           formPipette.pipetteName &&
-          formPipette.tiprackDefURI &&
+          formPipette.tiprackDefURI != null &&
```

Comment:
nit

```suggestion
          formPipette.pipetteName != null &&
          formPipette.tiprackDefURI != null &&
```

then we can remove `formPipette &&`.